packages/now-cli/src/util/dev/router.ts

```diff
@@ -27,6 +32,7 @@ export function resolveRouteParameters(
 }
 
 export default async function(
+  nowConfig: NowConfig,
```

Comment:
Since `devServer` is already being passed in, can't we get the `nowConfig` from the `getNowConfig()` function?

Reply:
@TooTallNate I was thinking that might slow the router down if it is called for each route but I guess its cached. If you're okay with that, then I'll change it.

Reply:
I updated the code to use `devServer.getNowConfig()` like you suggested 👍 
I also removed the `Using cached now.json config` debug log output.